styles/raindrop/catppuccin.user.css

```diff
@@ -0,0 +1,176 @@
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    /* Start styling your website here: */
+    &,
+    [data-theme] {
+      --background-color: @base;
+      --disable-background-color: @crust;
+      --disable-text-color: @overlay1;
+      --accent-color: @accent-color;
+      [data-variant="active"] {
+        background: fade(@accent-color, 10%) !important;
+      }
+      --alternative-background-color: @crust;
+      --sidebar-background-color: @mantle;
+      --hover-background-color: fade(@overlay0, 20%);
+      --active-background-color: @surface1;
+      --primary-text-color: @text;
+      --secondary-text-color: @subtext0;
+      --shadow-color: fade(@text, 10%);
+      --shadow-light-color: fade(@text, 10%);
+      // bookmark edit page header
+      [class^="header-"][data-fancy="true"] {
+        background: linear-gradient(
+          to bottom,
+          var(--background-color) 30%,
+          fade(@base, 60%) 100%
+        );
+      }
+      // bookmark edit page cover selector
+      [class^="cover-"] > [class^="more-"] {
+        background: fade(@accent-color, 85%);
+      }
+      // add bookmark button text color
+      [title="Add Bookmark"] {
+        color: @base
+      }
+      // disabled add bookmark button
+      [class^="button-"][data-variant="primary"][disabled] {
+        filter: none;
+        opacity: 100%;
+        background: @overlay0;
+        color: @crust;
+      }
+      // dropping bookmark to collection color overlay
+      [class*="isDropping-"] {
+        background-color: fade(@accent-color, 10%);
+      }
+      // sidebar background color
+      .svSidebar {
+        background: var(--sidebar-background-color);
+      }
+      // colors
+      color: @text;
+      --danger-color: @red;
+      --important-color: @red;
+      --success-color: @green;
+      --highlights-color: @mauve;
+      --reminder-color: @peach;
+      // filters
+      --note-color: @yellow;
+      --article-color: @peach;
+      --audio-color: @mauve;
+      --document-color: @subtext0;
+      --book-color: @rosewater;
+      --image-color: @green;
+      --video-color: @blue;
+      --broken-color: @maroon;
+      --duplicate-color: @sapphire;
+      --tag-color: @subtext0;
+      --filter-color: @subtext0;
```

Comment:
Generally me and other userstyle authors like to organize by placing all of the variables at the "top" of the userstyle and custom adjustments nearer to the bottom. Might help here instead of having intermingling CSS selectors with the vars.